packages/mui-system/src/breakpoints/breakpoints.js

```diff
@@ -1,7 +1,12 @@
 import PropTypes from 'prop-types';
+import isObjectEmpty from '@mui/utils/isObjectEmpty';
+import fastDeepAssign from '@mui/utils/fastDeepAssign';
```

Comment:
```suggestion
import deepAssign from '@mui/utils/deepAssign';
```

Reply:
When I name something `fast...` it usually means there's a trade-off with correctness to focus on being "fast". Here it will be that it doesn't handle exotic objects as other implementations will, as in, it will also traverse enumerable properties on the object's prototype. In the cases where we use this function (to create React props objects copies, or to create CSS style objects) we can safely assume that invariant is true. I should add this to the jsdoc, but I like having some sort of warning in the name.

Reply:
Adding this to the jsdom would definitely help. On the other hand, I would recommend maybe moving this util outside of @mui/utils, if it is only used in the mui-system, to avoid developers accidently using it in other places where it may not be the right thing. If you already plan to use it in other places, then probably we can keep it in @mui/utils.